<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfflineStorageConfigDialog</class>
 <widget class="QDialog" name="OfflineStorageConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Offline Storage Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure offline storage&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line17">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="databaseGroup">
     <property name="title">
      <string>Web SQL Databases</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="databaseEnabledCheckBox">
        <property name="toolTip">
         <string>Select to enable Web SQL Databases</string>
        </property>
        <property name="text">
         <string>Enable Web SQL Databases</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Default Quota:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="databaseQuotaSpinBox">
        <property name="toolTip">
         <string>Enter the default quota for Web SQL Databases</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> MB</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>306</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="showDatabasesButton">
        <property name="toolTip">
         <string>Press to show the Web SQL Databases</string>
        </property>
        <property name="text">
         <string>Show databases...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="applicationCacheGroup">
     <property name="title">
      <string>Offline Web Application Cache</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="applicationCacheEnabledCheckBox">
        <property name="toolTip">
         <string>Select to enable Offline Web Application Caches</string>
        </property>
        <property name="text">
         <string>Enable Offline Web Application Caches</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Default Quota:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="applicationCacheQuotaSpinBox">
        <property name="toolTip">
         <string>Enter the default quota for Offline Web Application Caches</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> MB</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>288</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="localStorageGroup">
     <property name="title">
      <string>Local Web Storage</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="localStorageEnabledCheckBox">
        <property name="toolTip">
         <string>Select to enable Local Web Storage</string>
        </property>
        <property name="text">
         <string>Enable Local Web Storage</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="localRemoteUrlsCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select to allow local content to access remote URLs</string>
        </property>
        <property name="text">
         <string>Local Content can access Remote URLs</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="localFileUrlsCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select to allow local content to access local files</string>
        </property>
        <property name="text">
         <string>Local Content can access Local Files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>databaseEnabledCheckBox</tabstop>
  <tabstop>databaseQuotaSpinBox</tabstop>
  <tabstop>showDatabasesButton</tabstop>
  <tabstop>applicationCacheEnabledCheckBox</tabstop>
  <tabstop>applicationCacheQuotaSpinBox</tabstop>
  <tabstop>localStorageEnabledCheckBox</tabstop>
  <tabstop>localRemoteUrlsCheckBox</tabstop>
  <tabstop>localFileUrlsCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OfflineStorageConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OfflineStorageConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localStorageEnabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>localRemoteUrlsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localStorageEnabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>localFileUrlsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
